File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.c
#include <stdio.h>
#include <string.h>

int main(void) {
	int n, total = 0;
	char arr[101];
	scanf("%d", &n);
	
	while (n--) {
		scanf("%s", arr);
		int hap = 1;
		for (int i = 0; i < strlen(arr); i++) {
			for (int j = i + 1; j < strlen(arr); j++) {
				if (arr[i] == arr[j]) {
					if (j - i >= 2) {
						if (arr[i] != arr[j - 1]) {
							hap = 0;
						}
					}
				}
			}
		}
		if (hap) total++;
	}
	printf("%d", total);
	return 0; 
}

File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.py
num = int(input())
total = 0

while(num > 0):
    arr = list(input())
    length = len(arr)
    hap = 0
    for i in range(length):
        for j in range(i + 1, length):
            if(arr[i] == arr[j]):
                sub = j - i
                if(sub >= 2):
                    if(arr[i] != arr[j - 1]): hap += 1
    if(hap == 0): total += 1
    num -= 1

print(total)